src/site/content/en/spaces/space-x1-test/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Title Goes Here Title Goes Here Title Goes Here
+description: We’re learning all about Vertex AI this week as Carter Morgan and Jay Jenkins Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+tags: twitter-space
+hosts:
+  - name: Name 1 here
+    twitter: devnook
+    image: https://pbs.twimg.com/profile_images/519869363344515072/s8vGg8fE_400x400.jpeg
```

Comment:
I know that this is just test data, but what's the strategy for displaying Twitter profile images once we have live data?

Hotlinking to hardcoded `pbs.twimg.com` URLs seems like it could be problematic, both because that `400x400.jpeg` URL doesn't seem to be officially supported, and because someone might change their Twitter profile image at some point and that might break existing URLs.

What would the process be for asking folks to provide Twitter profile image URLs when they wanted to create a new event?

It might be safer to rely on the profile images we already have for authors, served from URLs we control, even if their web.dev profile image differs from their Twitter profile image.

Here's Twitter guidance: https://developer.twitter.com/en/docs/twitter-api/v1/accounts-and-users/user-profile-images-and-banners

Reply:
Spaces hosts will not necessarily be web.dev authors, this is why we are not using author's images for this avatars. From technical point of view, this can be any img url. I'll work with Spaces creators to get a process established for those pics (e.g. uploading copy to our img pipeline).

Reply:
+1 to using the existing images we have and using web.dev authors yaml here. Can we still update this and go with this approach?